book/visual-effects.md

```diff
@@ -1202,8 +1203,8 @@ class Pixel:
 I want to emphasize that this code is not a part of our browser---I'm
 simply using Python code to illustrate what Skia is doing internally.
 
-That `Alphaf` operation applies to pixels in one surface. But with
-`SaveLayer` we will end up with two surfaces, with all of their pixels
+That `alphaf` operation applies to pixels in one surface. But with
```

Comment:
```suggestion
That `Alphaf` operation applies to pixels in one surface. But with
```

Reply:
Ah, I see, you meant the Skia parameter name. Got it.